### DISCUSSION TITLE
Local production build test

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [neonexus/sails-react-bootstrap-webpack#80] [General]
Hi @neonexus ,

This may be a noob question but it would be very much helpful for me if you could explain a bit.

For the production site, I understand that the `npm run build` would produce all the front-end bundle while `npm run lift:prod` will lift the sails for production.

But how do I see the front-end on production build?

I am not sure how the sails connect with bundled front-end and how do see the website in production? 

Sorry, I know this is a basic question but this is first big web application project as I mainly work on backend services. If you could kindly clarify how this works, it would be very much appreacited.

Thanks.

**Comment by neonexus** [comment]
@MaheshkumarSundaram if you still have this section: https://github.com/neonexus/sails-react-bootstrap-webpack/blob/release/config/routes.js#L23 in your repo, then Sails will automatically serve up whatever bundle. 

So, in my example repo, I use `main` and `admin` as 2 different React bundles. These get built in `.tmp/public`, in their respective folders. If you go to your prod instance of Sails (something like `https://myserver.com/`), it should automatically kick the user to `/main`, which will then tell Sails to serve the `main` bundle (stored in `.tmp/public/main`). If you went to `https://myserver.com/admin`, then Sails would serve the `admin` bundle (`.tmp/public/admin`). Sails only uses the first part of the URL to determine which bundle to use. This is why all the API calls are prefixed with `/api`.

**Reply by MaheshkumarSundaram** [reply]
Great. I missed the part about `routes.js`. Thank you so much for your explanation.

**Comment by neonexus** [comment]
Re-opening this discussion for ease of viewing.
